fairlearn/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class UnfairDataWarning(UserWarning):
+    """Custom warning meant to be raised when loading unfair datasets."""
```

Comment:
A small language tweak: a dataset is not unfair, it has "fairness issues". Suggestions:
* `UnfairDataWarning` -> `DataFairnessWarning`
* docstring : "... when loading datasets with fairness issues."